horovod/run/mpi_run.py

```diff
@@ -76,26 +86,39 @@ def mpi_run(settings, common_intfs, env):
     nccl_socket_intf_arg = '-x NCCL_SOCKET_IFNAME={common_intfs}'.format(
         common_intfs=','.join(common_intfs)) if common_intfs else ''
 
+    # Pick MPI implementation-specific flags
+    if mpi_impl == _MPI_IMPLS.SMPI:
+        mpi_args = _SMPI_FLAGS
+    else:
+        mpi_args = _OMPI_FLAGS
+
+    # On large cluster runs (e.g. Summit), we need extra settings to work around OpenMPI issues
+    if settings.num_hosts >= 64:
```

Comment:
This feels like it should be a configurable arg to `horovodrun`.  Maybe `--large-cluster-hosts-threshold` with a default of 64?

Reply:
I am hopping some of it get resolved in coming OpenMPI versions. We have seen those issues reported on 64 nodes or larger runs but never under that (related to the total number of daemons MPI needs to start on the boxes). So, it is a bootstrapping issue of MPI.
@tgaddair What is the advantage to have an argument for the user to pass it in? I am afraid if it is not useful, it might just be confusing.

Reply:
To be clear, those flags only impact the bootstrapping of the MPI processes but there is no impact on the processes once they run.

Reply:
I see, thanks for clarifying.  My concern was that 64 seemed somewhat empirical and possibly implementation specific, and I wouldn't want a user to run into unexpected behavior when they scale up to 64 nodes.  But if it's a known issue with MPI that specifically affects jobs with greater than 64 nodes, then I think it's fine to hard-code it.